templates/launch.html

```diff
@@ -177,6 +189,19 @@ <h3>${survey_type} Survey Metadata</h3>
         `
     }
 
+    function clearSchemaNameSelection() {
+        let schemaNameElement = document.querySelector("#schema_name")
+        schemaNameElement.selectedIndex = 0
+
+        let launchPattern = document.querySelector("#launch_pattern").value
+
+        if (launchPattern === "v1") {
+            document.querySelectorAll(".Social").forEach(function(element) {element.setAttribute("disabled", "disabled")})
+        } else {
+            document.querySelectorAll(".Social").forEach(function(element) {element.removeAttribute("disabled")})
+        }
```

Comment:
```suggestion
        }
        
        const schemaName = schemaNameElement.value

        if (schemaName !== "Select Schema" && launchPattern === "v1") {
            document.querySelector("#submit-btn-v1").disabled = false;
            document.querySelector("#submit-btn-v2").disabled = true;
        }
        else if (schemaName !== "Select Schema" && launchPattern === "v2") {
            document.querySelector("#submit-btn-v2").disabled = false;
            document.querySelector("#submit-btn-v1").disabled = true;
        }
        else {
            document.querySelector("#submit-btn-v2").disabled = true;
            document.querySelector("#submit-btn-v1").disabled = true;
        }
```

As we discussed earlier, both buttons must disappear when no schema is selected after you change the launch pattern version.